app/controllers/repairs_controller.rb

```diff
@@ -2,86 +2,46 @@ class RepairsController < ApplicationController
   before_filter :authenticate_user!
   load_and_authorize_resource
 
-  # GET /repairs
-  # GET /repairs.json
   def index
     @repairs = Repair.all
-
-    respond_to do |format|
-      format.html # index.html.erb
-      format.json { render json: @repairs }
-    end
   end
 
-  # GET /repairs/1
-  # GET /repairs/1.json
   def show
-    @repair = Repair.find(params[:id])
-
-    respond_to do |format|
-      format.html # show.html.erb
-      format.json { render json: @repair }
-    end
+    @last_editor = last_editor(@repair)
+    @page_title = "Repair for #{@repair.item}"
   end
 
-  # GET /repairs/new
-  # GET /repairs/new.json
   def new
     @repair = Repair.new
     @repair.item_id = params[:item_id].presence
-
-    respond_to do |format|
-      format.html # new.html.erb
-      format.json { render json: @repair }
-    end
   end
 
-  # GET /repairs/1/edit
   def edit
-    @repair = Repair.find(params[:id])
   end
 
-  # POST /repairs
-  # POST /repairs.json
   def create
     @repair = Repair.new(params[:repair])
 
     respond_to do |format|
       if @repair.save
-        format.html { redirect_to @repair, notice: 'Repair was successfully created.' }
-        format.json { render json: @repair, status: :created, location: @repair }
+        redirect_to @repair, notice: 'Repair was successfully created.'
       else
-        format.html { render action: "new" }
-        format.json { render json: @repair.errors, status: :unprocessable_entity }
+        render action: "new"
```

Comment:
if the repair failed to save shouldn't there be an error message saying why? No rails flash?